guidance/models/_model.py

```diff
@@ -835,11 +855,19 @@ def _cleanup_tokens(self, token_ids, token_byte_positions):
             ):
                 for i in range(1, len(token_byte_positions)):
                     token_byte_positions[i] -= 1
-            
+
             # another ugly hack for tokenizers that are not stable on encode/decode cycles
             # currently only Phi-3, should generalize this method if we see more of these
-            if not hasattr(self, "_disable_retokenize_check"):
-                assert token_byte_positions[-1] == last_pos, "Cross check last_pos"
+            if token_byte_positions[-1] != last_pos:
```

Comment:
This is the actual change. The rest of this file is what happens when you don't enforce linting.

Reply:
I think almost every change `black` made here is _significantly_ less readable :/. We should spend some time investigating black settings to see if there's a way to 1) handle trailing line comments and 2) perhaps a wider line width setting, but I don't think we should dogmatically enforce black linting for the time being until we resolve it (separately from this PR).

Reply:
Thanks for pointing this out @riedgar-ms

Reply:
`black` has very few settings; the line length being one of the only ones, which we set in `pyproject.toml`. It is meant o be opinionated, and it just doesn't seem to like trailing comments. One alternative which appears to be gaining some popularity is `ruff`:
https://docs.astral.sh/ruff/formatter/#configuration